packages/shared/__tests__/setup.ts

```diff
@@ -7,8 +7,6 @@ import { storageWrapper as storage } from '../src/lib/storageWrapper';
 
 process.env.NEXT_PUBLIC_API_URL = 'http://localhost:3000';
 process.env.NEXT_PUBLIC_WEBAPP_URL = '/';
-// eslint-disable-next-line @typescript-eslint/no-unused-vars, @typescript-eslint/no-empty-function
-window.ga = ((...args) => {}) as UniversalAnalytics.ga;
```

Comment:
Why remove? or just not needed in tests anymore?

Reply:
It's not used anywhere the non test version was commented out so also deleted that as it might trigger the review issue.